### ISSUE TITLE
CS8344 doesn't have a target page

### ISSUE THREAD

**Issue by richlander** [dotnet/docs#41075]
This link doesn't seem to work: https://msdn.microsoft.com/query/roslyn.query?appId=roslyn&k=k(CS8344)

I end up here: https://learn.microsoft.com/en-us/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error?f1url=%3FappId%3Droslyn%26k%3Dk(CS8344)

From C# Dev Kit.

<img width="486" alt="image" src="https://github.com/dotnet/docs/assets/2608468/95f732c5-55a4-4366-ba29-61f7a65ca71f">

**Comment by richlander** [comment]
Yup. No issue with the actual error. It was correct and I was able to act upon it. This is solely about docs.

**Comment by BartoszKlonowski** [comment]
I know it's been a while since you raised the topic, but maybe you can still share a minimal example of how you solved your problem? Maybe I can use that as a valid example of the resolution (as real examples are always more intuitive).

**Comment by richlander** [comment]
That's a good idea/request. I know the general codebase I was working on. I don't quite remember the exact thing I was doing. When I'm next working on it, I'll see if I can come up with a pattern.

**Comment by BartoszKlonowski** [comment]
@richlander I can see the link you initially reported now contains the `CS8344` - are we good to close this issue?
CC: @BillWagner
